### DISCUSSION TITLE
✅ 🔗 Custom Post Loop Templates (Duplicate of #4440)

### DISCUSSION THREAD

**Discussion by toltechinternet** [elementor/elementor#19050] [Feature Request]
Hi,

It would be great to see Elementor implementing a "Loop" section to their theme build which allows us to target different post types and output loops using their posts widget. There is an existing plugin which achieves this result by the name of "Elementor Custom Skin". This plugin hooks into the posts widget and allows you to set the parameters of which post type and what template you would like to use.

Usually, if there is an existing plugin which achieves the desired outcome, Elementor wouldn't consider the feature. However, this plugin is poorly supported and their staff have not responded to any support requests from users.

This feature would bring a huge range of additional possibilities on how the Elementor community would target their post types and output the information.

**Comment by arielk** [comment]
Duplicate https://github.com/elementor/elementor/issues/4440

**Comment by nicholaszein** [comment]
Hello there!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with the new Loop Builder in Elementor Pro v3.8! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
